pkg/nsx/util/utils.go

```diff
@@ -369,3 +372,29 @@ func castApiError(apiErrorDataValue *data.StructValue) *model.ApiError {
 func isEmptyAPIError(apiError model.ApiError) bool {
 	return (apiError.ErrorCode == nil && apiError.ErrorMessage == nil)
 }
+
+func VerifyNsxCertWithThumbprint(der []byte, thumbprint string) error {
```

Comment:
curious, what does the name der stand for?

Reply:
> DER (Distinguished Encoding Rules) is a binary encoding for [X.509](https://www.ssl.com/faqs/what-is-an-x-509-certificate/) certificates and private keys.

Under our context it's the binary (byte) encoded form of a cert